src/libraries/System.Text.Json/tests/Common/PropertyNameTests.cs

```diff
@@ -494,5 +494,31 @@ public class ClassWithSpecialCharacters
             [JsonPropertyName("\uA000_2")] // Valid C# property name: \uA000_2
             public int YiIt_2 { get; set; }
         }
+
+        [Fact]
```

Comment:
>Fact [](http://example.com/codeflow?start=9&length=4)

nit: maybe better convert this method to Theory and passing Boolean parameter that set `options.PropertyNameCaseInsensitive`. This will allow testing both case sensitive and insensitive.

Reply:
Addressed.